frontend/components/queries/queryResults/AwaitingResults/AwaitingResults.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import PhoneHome from "../../../../../assets/images/phone-home.svg";
+
+const baseClass = "awaiting-results";
+
+const AwaitingResults = () => {
+  return (
+    <div className={baseClass}>
+      <img src={PhoneHome} alt="awaiting results" />
+      <span className={`${baseClass}__title`}>Phoning home...</span>
+      <p className={`${baseClass}__description`}>
+        There are currently no results to your query. Please wait while we talk
```

Comment:
The design for these 2 sentences is two separate lines and centered in the UI

Reply:
@ghernandez345

Reply:
Im letting the width of the container determine line breaks. Since the width and the text size is the exact values that are in figma the lines will break correctly without the need for <br>.

I have added the centering now.